### PR TITLE
Feat implement semantic publish

### PR DESCRIPTION
**Pull request by ManuelJrez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ We build this project highly inspired in `create-obytes-app` and `ignite-templat
 You can `cd APP_NAME` and try to run your app using the following command:
 
 `npm run ios:dev || npm run android:dev`
+
+sdlkjfsjld
```
Comment (Stukz) [inline README.md:18]: I think you forgot to remove this

**Review by Stukz — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ManuelJrez) [reply README.md:18]: why you think that? i think it's okay because is the way that we could run our base, right? 🤔 @Stukz

Comment (aguscha333) [reply README.md:18]: haha approving once this is removed

**Review by aguscha333 — COMMENTED** [review]
@ManuelJrez I see that the name of the package is still `create-rootstrap-app`. It'd be good to change it before publishing

**Review by aguscha333 — APPROVED** [review]

**Review by asdolo — APPROVED** [review]

**Pull request merged**